maintainers/maintainer-list.nix

```diff
@@ -3071,6 +3071,12 @@
     githubId = 1276854;
     name = "Florian Peter";
   };
+  fbrs = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add fbrs``.

Reply:
Done